spec/support/ddf/valid_service_plan_ddf.json

```diff
@@ -91,10 +91,64 @@
             "value": 10
           }
         ],
-        "component": "text-field",
+        "component": "textarea-field",
         "helperText": "",
         "isRequired": true,
         "initialValue": 5
+      },
+      {
+        "name": "date picker",
+        "label": "Date Picker",
+        "component": "date-picker"
+      },
```

Comment:
@lindgrenj6 were we going to add the rest of the field types to the openapi.json spec as well?

Reply:
I hadn't thought of that but it's a good idea, I'll add it.